src/serial/plugin-PixelTriplets/CAHitNtupletGeneratorKernelsImpl.h

```diff
@@ -188,31 +188,31 @@ __global__ void kernel_fastDuplicateRemover(GPUCACell const *__restrict__ cells,
   }
 }
 
-__global__ void kernel_connect(cms::cuda::AtomicPairCounter *apc1,
-                               cms::cuda::AtomicPairCounter *apc2,  // just to zero them,
-                               GPUCACell::Hits const *__restrict__ hhp,
-                               GPUCACell *cells,
-                               uint32_t const *__restrict__ nCells,
-                               gpuPixelDoublets::CellNeighborsVector *cellNeighbors,
-                               GPUCACell::OuterHitOfCell const *__restrict__ isOuterHitOfCell,
-                               float hardCurvCut,
-                               float ptmin,
-                               float CAThetaCutBarrel,
-                               float CAThetaCutForward,
-                               float dcaCutInnerTriplet,
-                               float dcaCutOuterTriplet) {
+void kernel_connect(cms::cuda::AtomicPairCounter *apc1,
+                    cms::cuda::AtomicPairCounter *apc2,  // just to zero them,
+                    GPUCACell::Hits const *__restrict__ hhp,
+                    GPUCACell *cells,
+                    uint32_t const *__restrict__ nCells,
+                    gpuPixelDoublets::CellNeighborsVector *cellNeighbors,
+                    GPUCACell::OuterHitOfCell const *__restrict__ isOuterHitOfCell,
+                    float hardCurvCut,
+                    float ptmin,
+                    float CAThetaCutBarrel,
+                    float CAThetaCutForward,
+                    float dcaCutInnerTriplet,
+                    float dcaCutOuterTriplet) {
   auto const &hh = *hhp;
 
-  auto firstCellIndex = threadIdx.y + blockIdx.y * blockDim.y;
-  auto first = threadIdx.x;
-  auto stride = blockDim.x;
+  auto firstCellIndex = 0 + 0 * 1;
```

Comment:
Could be simplified to
```suggestion
  auto firstCellIndex = 0;
```
?